src/main/resources/ath-container/Dockerfile

```diff
@@ -22,8 +22,11 @@ RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive \
     TZ=Etc/UTC \
     apt-get install -y --no-install-recommends \
+        # docker dep
+        ca-certificates \
+        gnupg \
+        lsb-release \
         curl \
-        docker \
```

Comment:
So.. isn't this a case of just using Ng the right package in Ubuntu?
https://packages.ubuntu.com/jammy/amd64/docker.io/filelist

If we wanted the latest docker-ce then the scripts are fine, but I think we can still use apt can't we?

Reply:
The « repackaged » docker for ubuntu had been a pain for years on the productions we managed. It kinda shift quite often from official docker ce (not mentioning missing buildx, time to have cve fixed etc.). 
Using the docker-ce package is « only » adding a few lines here, but for the better.

Reply:
And in many cases you get a too old version of docker. Using the official repos from docker makes it easier to update versions if needed plus contains the most recent versions and fixes if needed.